pkg/recovery/helpers.go

```diff
@@ -92,7 +93,7 @@ func EnsureFileContent(filePath string, data []byte) error {
 			return fmt.Errorf("file %q is a directory", filePath)
 		}
 
-		fileBytes, err := ioutil.ReadFile(filePath)
+		fileBytes, err := ioutil.ReadFile(filepath.Clean(filePath))
```

Comment:
why does this contribute to security?

Reply:
The intent is to sanitize file paths that might come from the user. If this is useless, we can tag it as such with the comment:

`// #nosec`

Reply:
in this case we sanitize paths which are passed to a Linux system call. Don't think this adds anything. Linux will do the same logic.